### DISCUSSION TITLE
Find Game screens

### DISCUSSION THREAD

**Discussion by andymarden** [online-go/online-go.com#2888] [Polls]
Not a fan of the new screens when I click "find game"  and I want to create the game I normally do (on Android mobile web). I find it very cluttered with too much on each screen

I felt the original was better than what we have now but  perhaps that could be improved. 

So a poll of three options:

**Comment by BHydden** [comment]
anoek asked you to discuss this on the forum https://forums.online-go.com/ you're not likely to get much engagement here.

**Reply by andymarden** [reply]
Oh god yes - brain malfunction (not well at the moment so that's my excuse). Will do.
